### ISSUE TITLE
SQL server database backup retention policies are unsupported

### ISSUE THREAD

**Issue by baskar-natarajan** [pulumi/pulumi-azure-native#755]
Converting the following in `arm2pulumi`

```
{
"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
"contentVersion": "1.0.0.0",
"parameters": {
    "serverName": {
        "type": "string"
    },
    "databaseName": {
        "type": "string"
    },
    "longTermWeeklyRetentionPeriod": {
        "type": "string"
    },
    "pointInTimeRetentionDays": {
        "type": "string"
    }
},
"resources": [
    {
        "type": "Microsoft.Sql/servers/databases/backupLongTermRetentionPolicies",
        "name": "[concat(parameters('serverName'), '/', parameters('databaseName'), '/Default')]",
        "apiVersion": "2020-11-01-preview",
        "properties": {
            "weeklyRetention": "[parameters('longTermWeeklyRetentionPeriod')]"
        }
    },
    {
        "type": "Microsoft.Sql/servers/databases/backupShortTermRetentionPolicies",
        "name": "[concat(parameters('serverName'), '/', parameters('databaseName'), '/Default')]",
        "apiVersion": "2020-11-01-preview",
        "properties": {
            "retentionDays": "[parameters('pointInTimeRetentionDays')]"
        }
    }]
}
```

results in 

```
using Pulumi;

class MyStack : Stack
{
    public MyStack()
    {
        var config = new Config();
        var databaseNameParam = config.Require("databaseNameParam");
        var longTermWeeklyRetentionPeriodParam = config.Require("longTermWeeklyRetentionPeriodParam");
        var serverNameParam = config.Require("serverNameParam");
        var backupLongTermRetentionPolicyResource = null;
        var pointInTimeRetentionDaysParam = config.Require("pointInTimeRetentionDaysParam");
        var backupShortTermRetentionPolicyResource = null;
    }

}
```

and the following error:

```
backupLongTermRetentionPolicyResource  no metadata found for resource type 'Microsoft.Sql/servers/databases/backupLongTermRetentionPolicies' and version '2020-11-01-preview'
backupShortTermRetentionPolicyResource  no metadata found for resource type 'Microsoft.Sql/servers/databases/backupShortTermRetentionPolicies' and version '2020-11-01-preview'
```

But the 2020-11-01-preview version is valid for backup long term & short term retention policies.

**Comment by mikhailshilkov** [comment]
@viveklak What's our process for updating arm2pulumi to latest versions? These resources have just been released in 0.9.0.

**Comment by viveklak** [comment]
This has now been updated to reflect the 1.1.0 release.
